lib/receiver.js

```diff
@@ -356,7 +356,7 @@ class Receiver extends Writable {
       );
     }
 
-    this._payloadLength = num * Math.pow(2, 32) + buf.readUInt32BE(4);
+    this._payloadLength = buf.readUIntBE(2, 6);
```

Comment:
This actually reduces accuracy from 53 bits to 48 bits.

Reply:
So, the prior existing math is correct in that it accounts for 64 bits of addressable space after erroring if the payload size is greater than the (2**53) - 1.  Maybe for a future pull request I could submit something like `this._payloadLength = [num, buf.readUInt32BE(4)];` similar to the orientation Node uses for the now deprecated process.hrtime.  That would also be cleaner than accounting for the 53bit limitation, but its a larger refactor than the subject of this pull request.

I have found in my own web socket implementation the payload length of the receiving payload is more important than I originally thought because sometimes include trailing noise on the fit packet that needs to be trimmed.